crates/ruff_linter/resources/test/fixtures/ruff/RUF052.py

```diff
@@ -1,4 +1,4 @@
-# Correct
+##################### Correct #####################
```

Comment:
This might be just personal taste, so happy to remove!

But when I saw just `# Correct`, I wasn't sure if that was demarcating a section of code.  I personally find this more clear.